tests/test_tools.py

```diff
@@ -15,7 +15,7 @@ def test_up_down_bias():
 
 def test_chl():
     ds = fetchers.load_sample_dataset()
-    tools.chl_first_check(ds)
+    tools.optics_first_check(ds)
```

Comment:
add some more test cases here! Now that the function is more flexible, it should get more tests